### ISSUE TITLE
[1][gibbsfromncis] Do not include hidden assets in Overall Balance

### ISSUE THREAD

**Issue by wmbutler** [bitshares/bitshares-ui#1061]
When I hide assets, it's generally because they were sent to me or they have little value to me. Right now, hidden assets are factored into my net worth totals. This was fine until recently when the PEERPLAYS asset (used for their ICO) suddenly increased in value to 1,991.0007 per share. This is effectively skewing my balance by 348,414.

I think it's reasonable to exclude hidden assets from the total to deal with situations like this...thoughts? If you disagree, please post other ideas.

<img width="1267" alt="screen shot 2018-01-23 at 6 22 56 pm" src="https://user-images.githubusercontent.com/1254810/35308170-dfc42ea0-006b-11e8-91c0-f389ca9720f3.png">

**Comment by ahdigital** [comment]
I don't think hidden assets should be removed from the users total net worth unless it can be a user controllable setting that can be toggled on/off.

In the PEERPLAYS example, this is a positive experience for the user which otherwise might have gone unnoticed if it was hidden from view and net worth.

Incidently, I personally have never used the hide assets functionality.

What I think is more confusing for the user is that it's not clear the net worth in the top right of the UI is combined assets and not just the amount of BTS you have. I was sent BOOZECOIN and didnt exchange it for BTS for a while because it looked like it had already been exchanged. It needs to speak 'combined net worth' to the user if that is what it is.

**Comment by startailcoon** [comment]
I agree with hidden assets should not be counted in the total wallet value. I have a few tokens that I've received over time. These doesn't have a market generally but at times someone sell/buy and I suddenly get an increase of "fake" USD value.

Somehow we need the ability to hide these assets proposed value for the wallet.

**Comment by HarukaMa** [comment]
Maybe provide a settings entry to control if hidden assets value should be counted into total value?

**Comment by wmbutler** [comment]
@ahdigital technically it's not a positive experience because that valuation is not true. It was likely a dust trade a high price that anyone could fake in any market.

**Comment by ahdigital** [comment]
@wmbutler I'm not 100% sure I understand what a dust trade is and imagine that to be the case for many users.

If your example (and others) generally result in a negative user experience perhaps they should be excluded from the net worth to be on the safe side.

**Comment by wmbutler** [comment]
@ahdigital Just as an example, if you owned an asset that was essentially worthless, you could post .00000001 of that asset for sale at $1,000,000. Then you could buy it for $1 and that would be the last trade on the books. The PEERPLAYS asset is particularly annoying because it is blacklisted for trading or transferring. So, I'm stuck with a skewed balance with no way to change it.

**Comment by tbone-bts** [comment]
I agree with excluding hidden asset balances from net worth total.  But it would be helpful to show the total value of hidden assets at the bottom of the hidden assets view (just as there's a total value of active assets at the bottom of the active assets view).

**Comment by wmbutler** [comment]
- [ ] Deduct hidden balances from portfolio balance
- [ ] Include a Total at the summary of hidden balances.

**Comment by wmbutler** [comment]
@gibbsfromncis were you planning to work this one?

**Comment by gibbsfromncis** [comment]
@wmbutler yes sure. I have some busy days and tomorrow I'll start to work on tasks for this Sprint.

First of all I'll close the task about QR libraries to unblock startailcoon and then I'm going to start work on this task. My work flow looks pretty simple - close blocking tasks, fix bugs, work on important features, work on low priority features :)

**Comment by gibbsfromncis** [comment]
@wmbutler btw I don't take tasks I'm not able to work on this Sprint. I'm trying to understand priority of all features/bugs and work from high priority issues to low. if I see I'm not able to work on something/don't have a time/I don't understand how to fix etc. I'll notify you asap so anyone else be able to work on issue during this Sprint.

**Comment by gibbsfromncis** [comment]
@wmbutler this should make you happy today 😃 PR: #1126

**Comment by wmbutler** [comment]
Thanks!! So happy to see this complete. I think this will be a welcomed addition for many users. Regarding priority, anything tagged as a bug is top priority. I also will sometimes label features as high priority.

**Comment by wmbutler** [comment]
@gibbsfromncis can you also deduct the hidden balances from the overall total in the top right? As you can see in this screenshot, Portfolio + Margin should equal the total in the upper right.

<img width="884" alt="screen shot 2018-02-11 at 12 29 08 pm" src="https://user-images.githubusercontent.com/1254810/36076894-5d3f27ac-0f28-11e8-893d-a10362b37e0d.png">

**Comment by gibbsfromncis** [comment]
@wmbutler sure, will work on it a little bit later.

**Comment by gibbsfromncis** [comment]
@wmbutler Done. PR #1166
